pages/index.js

```diff
@@ -20,45 +20,128 @@ export default function Home(props) {
         <title>{t("siteTitle")}</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <div className="bg-gray-light-200 py-6">
-        <div className="layout-container flex">
-          <ActionButton
-            text={t("buttonStartTesting")}
-            secondary
-            className={"mr-4"}
-            dataCyButton={"start-testing-button"}
-          />
-          <ActionButton
-            text={t("buttonBecomeTester")}
-            dataCyButton={"become-tester-button"}
-          />
-        </div>
-      </div>
-
-      <section className="layout-container">
-        <h2>{t("aboutUsHeading")}</h2>
-        <p>{t("aboutUsContent")}</p>
-
-        <figure>Block</figure>
 
-        <figure>
-          Block
-          <figcaption>{t("figCaption")}</figcaption>
-        </figure>
+      <section className="layout-container mb-8 mt-12">
+        <div className="flex flex-col           lg:grid lg:grid-cols-2 lg:gap-24">
+          <div>
+            <h1>{t("experimentsAndExplorationTitle")}</h1>
+            <br></br>
+            <p className="text-sm md:text-p leading-normal text-left font-normal font-body">
+              {t("experimentsAndExploration-1/3")} <br></br> <br></br>
+              {t("experimentsAndExploration-2/3")}
+              <br></br>
+              <br></br> {t("experimentsAndExploration-3/3")}
+            </p>
+            <br></br>
+          </div>
+          <div className="lg:pt-32 mb-6 ">
+            <div className="border-l-4">
+              <p className="text-sm md:text-p pl-4 leading-normal text-left font-normal font-body">
+                {t("experimentsComment")}
+              </p>
+              <p className="text-sm md:text-p pt-4 pl-4 text-gray-500 leading-normal text-left font-normal font-body">
+                {" "}
+                — Kim Lee Kho
+              </p>
+            </div>
+          </div>
+        </div>
+        <ActionButton
+          className={"text-xs md:text-base"}
+          text={t("experimentsBtnTxt")}
+          secondary
+          dataCyButton={"see-the-experiements"}
+        />
       </section>
 
-      <section className="layout-container">
-        <h3>{t("contactUsHeading")}</h3>
-        <p>{t("contactUsContent")}</p>
-        <a href="#">john.doe@example.com</a>
-        <figure>Block</figure>
-      </section>
+      <section className="bg-footer-background-color  ">
+        <div className="flex flex-col-reverse pt-8  layout-container   xl:grid xl:grid-cols-2 xl:gap-8">
+          <div>
+            <div className="flex flex-row ">
+              <div
+                className="bg-circle-color flex-shrink-0 opacity-40 mr-4 mb-2 rounded-full h-32 w-32 flex items-center justify-center
+           text-white font-bold text-h1xl relative md:left-0 -left-14 "
```

Comment:
I don't think "-left-14" is a valid class no?

Reply:
not really sure I found it on tailwind. I used it to move the circle out out of the margin or it or it will show white outline from the layout-margin. If I used a half circle it will still show a border of white .